templates/pages/account/add-return.html

```diff
@@ -4,7 +4,7 @@
 <h1 class="page-heading">{{lang 'account.returns.new_return' }}</h1>
 {{> components/account/navigation account_page='returns'}}
 
-<main class="account account--fixed account--addReturn">
+<div class="account account--fixed account--addReturn">
```

Comment:
Should we use section instead of divs?

Reply:
as a [rule](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) section tag should have a heading inside. And I've switched on section whenever it is possible but prefer to use div in case like this. since here div is used more as a wrapper for styling